--- a/bash/PKGBUILD
+++ b/bash/PKGBUILD
@@ -13,7 +13,7 @@
 license=('GPL')
 url='http://www.gnu.org/software/bash/bash.html'
 groups=('base')
-backup=(etc/bash.bash{rc,_logout} etc/skel/.bash{rc,_profile,_logout})
+backup=(etc/bash/bash{rc,_logout} etc/skel/.bash{rc,_profile,_logout})
 depends=('readline>=7.0' 'glibc' 'ncurses')
 optdepends=('bash-completion: for tab completion')
 provides=('sh')
@@ -22,7 +22,9 @@
         dot.bash_profile
         dot.bash_logout
         system.bashrc
-        system.bash_logout)
+        system.bash_logout
+        artix.bashrc
+        functions.bashrc)
 validpgpkeys=('7C0135FB088AAF6C66C650B9BB5869F064EA74AB') # Chet Ramey

 if [[ $((10#${_patchlevel})) -gt 0 ]]; then
@@ -45,8 +47,8 @@

   _bashconfig=(-DDEFAULT_PATH_VALUE=\'\"/usr/local/sbin:/usr/local/bin:/usr/bin\"\'
                -DSTANDARD_UTILS_PATH=\'\"/usr/bin\"\'
-               -DSYS_BASHRC=\'\"/etc/bash.bashrc\"\'
-               -DSYS_BASH_LOGOUT=\'\"/etc/bash.bash_logout\"\'
+               -DSYS_BASHRC=\'\"/etc/bash/bashrc\"\'
+               -DSYS_BASH_LOGOUT=\'\"/etc/bash/bash_logout\"\'
                -DNON_INTERACTIVE_LOGIN_SHELLS)
   export CFLAGS="${CFLAGS} ${_bashconfig[@]}"

@@ -63,10 +65,14 @@
   make -C $pkgname-$_basever DESTDIR="$pkgdir" install
   ln -s bash "$pkgdir"/usr/bin/sh

-  install -dm755 "$pkgdir"/etc/skel/
+  install -dm755 "$pkgdir"/etc/{skel,bash/bashrc.d}/
   # system-wide configuration files
-  install -m644 system.bashrc $pkgdir/etc/bash.bashrc
-  install -m644 system.bash_logout "$pkgdir"/etc/bash.bash_logout
+  install -m644 system.bashrc $pkgdir/etc/bash/bashrc
+  install -m644 system.bash_logout "$pkgdir"/etc/bash/bash_logout
+
+  # artix bashrc
+  install -m644 artix.bashrc "$pkgdir"/etc/bash/bashrc.d/artix.bashrc
+  install -m644 functions.bashrc "$pkgdir"/etc/bash/bashrc.d/functions.bashrc

   # user configuration file skeletons
   install -m644 dot.bashrc "$pkgdir"/etc/skel/.bashrc
--- a/bash/dot.bashrc
+++ b/bash/dot.bashrc
@@ -5,5 +5,5 @@
 # If not running interactively, don't do anything
 [[ $- != *i* ]] && return

-alias ls='ls --color=auto'
-PS1='[\u@\h \W]\$ '
+# alias ls='ls --color=auto'
+# PS1='[\u@\h \W]\$ '
--- a/bash/system.bash_logout
+++ b/bash/system.bash_logout
@@ -1,3 +1,3 @@
 #
-# /etc/bash.bash_logout
+# /etc/bash/bash_logout
 #
--- a/bash/system.bashrc
+++ b/bash/system.bashrc
@@ -1,5 +1,5 @@
 #
-# /etc/bash.bashrc
+# /etc/bash/bashrc
 #

 # If not running interactively, don't do anything
@@ -19,4 +19,8 @@
     ;;
 esac

+for sh in /etc/bash/bashrc.d/*.bashrc ; do
+	[[ -r ${sh} ]] && source "${sh}"
+done
+
 [ -r /usr/share/bash-completion/bash_completion   ] && . /usr/share/bash-completion/bash_completion

