--- a/bash/PKGBUILD
+++ b/bash/PKGBUILD
@@ -13,7 +13,7 @@
 license=('GPL')
 url='http://www.gnu.org/software/bash/bash.html'
 groups=('base')
-backup=(etc/bash.bash{rc,_logout} etc/skel/.bash{rc,_profile,_logout})
+backup=(etc/bash/bash{rc,_logout} etc/skel/.bash{rc,_profile,_logout})
 depends=('readline>=7.0' 'glibc' 'ncurses')
 optdepends=('bash-completion: for tab completion')
 provides=('sh')
@@ -22,7 +22,8 @@
         dot.bash_profile
         dot.bash_logout
         system.bashrc
-        system.bash_logout)
+        system.bash_logout
+        artix.bashrc)
 validpgpkeys=('7C0135FB088AAF6C66C650B9BB5869F064EA74AB') # Chet Ramey

 if [[ $((10#${_patchlevel})) -gt 0 ]]; then
@@ -45,8 +46,8 @@

   _bashconfig=(-DDEFAULT_PATH_VALUE=\'\"/usr/local/sbin:/usr/local/bin:/usr/bin\"\'
                -DSTANDARD_UTILS_PATH=\'\"/usr/bin\"\'
-               -DSYS_BASHRC=\'\"/etc/bash.bashrc\"\'
-               -DSYS_BASH_LOGOUT=\'\"/etc/bash.bash_logout\"\'
+               -DSYS_BASHRC=\'\"/etc/bash/bashrc\"\'
+               -DSYS_BASH_LOGOUT=\'\"/etc/bash/bash_logout\"\'
                -DNON_INTERACTIVE_LOGIN_SHELLS)
   export CFLAGS="${CFLAGS} ${_bashconfig[@]}"

@@ -63,10 +64,13 @@
   make -C $pkgname-$_basever DESTDIR="$pkgdir" install
   ln -s bash "$pkgdir"/usr/bin/sh

-  install -dm755 "$pkgdir"/etc/skel/
+  install -dm755 "$pkgdir"/etc/{skel,bash/bashrc.d}/
   # system-wide configuration files
-  install -m644 system.bashrc $pkgdir/etc/bash.bashrc
-  install -m644 system.bash_logout "$pkgdir"/etc/bash.bash_logout
+  install -m644 system.bashrc $pkgdir/etc/bash/bashrc
+  install -m644 system.bash_logout "$pkgdir"/etc/bash/bash_logout
+
+  # artix bashrc
+  install -m644 artix.bashrc "$pkgdir"/etc/bash/bashrc.d/artix.bashrc

   # user configuration file skeletons
   install -m644 dot.bashrc "$pkgdir"/etc/skel/.bashrc

